man/crystal.1

```diff
@@ -0,0 +1,234 @@
+.It Fl -release
+Turn on optimizations for the generated code, which are disabled by default.
+.It Fl s, -stats
+Print runtime statistics about the different compiler stages for the current build. Output time and used memory for each compiler process.
+.It Fl -single-module
+Generate a single LLVM module.
+.It Fl -threads
+Maximum number of threads to use for code generation. The default is 8 threads.
+.It Fl -target Ar TRIPLE
+Enable target triple; intended to use for cross-compilation. See llvm documentation for more information about target triple.
+.It Fl -verbose
+Display the commands executed by the system.
+.El
+
+.Pp
+.It
+.Cm deps
+.Op options
+.Ar <command>
+.Pp
+Alias for shards, the dependency manager for Crystal language.
+.Pp
+.It
+.Cm docs
+.Pp
+Generate documentation from comments using a subset of markdown. The output is saved in html format on the created doc/ folder. More information about documentation conventions can be found at https://crystal-lang.org/docs/conventions/documenting_code.html.
+.Pp
+.It Cm eval
+.Pp
+Evaluate code from arguments or, if no arguments are passed, from the standard input. Useful for experiments.
+.Pp
+.It
+.Cm play
+.Op options
+.Pp
+Starts the crystal playground server on port 8080, by default.
+.Pp
+Options:
+.Bl -tag -width "12345678" -compact
+.Pp
+.It Fl p Fl -port
+Run the playground on the specified port. Default is 8080.
+.It Fl b Ar HOST Fl -binding Ar HOST
+Bind the playground to the specified IP.
+.It Fl v Fl -verbose
+Display detailed information of the executed code.
+.It Fl h Fl -help
+Show a help message about play command options.
+.El
+.Pp
+.It
+.Cm run
+.Op options
+programfile
+--
+.Op arguments
+.Pp
+The default command. Compile and run program.
+.Pp
+Options:
+Same as the build options.
+.Pp
+.It
+.Cm spec
+.Pp
+Compile and run specs (in spec directory).
+.Pp
+.It
+.Cm tool
+.Op tool
+.Op switches
+programfile
+--
+.Op arguments
+.Pp
+Run a tool. The available tools are: context, format, hierarchy, implementations, and types.
+.Pp
+Tools:
+.Bl -tag -offset indent
+.It Cm context
+Show context for given location.
+.It Cm format
+Format project, directories and/or files with the coding style used in the standard library. You can use the
+.Fl -check
+flag to check whether the formatter would make any changes.
+.It Cm hierarchy
+Show hierarchy of types from file. Also show class and struct members, with type and size. Types can be filtered with a regex by using the
+.Fl e
+flag.
+.It Cm implementations
+Show implementations for a given call. Use
+.Fl -cursor
+to specify the cursor position. The format for the cursor position is file:line:column.
+.It Cm types
+Show type of main variables of file.
+.El
+.Pp
+.It Cm version, Fl -version, v
+.Pp
+Show version.
+.El
+.Sh SEE ALSO
+.Fn shards 1
+.Bl -hang -compact -width "https://github.com/crystal-lang/crystal/1234"
```

Comment:
It really doesn't display; it is the argument for width. The url itself is not important, just its length. I can change this, if you want.